README.md

```diff
@@ -50,7 +50,7 @@ As of this writing, the oldest version of QGIS we support is 3.16.x LTR. You
 can find downloads and instructions
 [here](https://qgis.org/en/site/forusers/download.html).
 
-After installing QGIS, [download QGreenland](http://www.qgreenland.org/explore)
```

Comment:
This `explore` endpoint still exists but provides outdated information. Can we remove it from the website?